documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Each Corporate Overlord will contribute an $300 initial capital contribution
+to BIFFUD within three months of achieving Corporate Overlord status.
+
+Additional capital contributions can be voted on and approved by the
+Corporate Overlords during a plotting session.
```

Comment:
Here's another reference to a generic vote. I know Section 5 is being codified now, but maybe at some point somewhere it should be said that any CO vote, if not otherwise specified, must be 2/3 of a quorum or something like that.